### PR TITLE
Autonomous - MERGE OR THE UNICORNS WILL STAB YOU WITH THEIR HORNS

### PR DESCRIPTION
**Pull request by classygirl44**
Pull Request Checklist:
- [x] You are able to understand what the code is doing on a basic level with the use of comments
- [x] Code is formatted in a way that isn't confusing
- [x] builds cleanly
- [x] builds *to the robot* cleanly
- [x] Code has been tested on the robot/roadkill/other mechanical setup and completes the task it needs to complete

### CODE REVIEW

File: src/main/java/frc/robot/RobotMercury.java [hunk 1]
```diff
@@ -84,8 +86,10 @@ public void robotInit() {
     turretControls = new TurretControls(turret);
     registerSubsystem((SubsystemBase)turret, turretControls);
 
-    widgets = new ShuffleboardWidgets();
-    widgets.register();
+    //widgets = new ShuffleboardWidgets();
```
Comment (r-wallis) [inline src/main/java/frc/robot/RobotMercury.java]: put widgets back

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -82,8 +83,10 @@ public void robotInit() {
     turretControls = new TurretControls(turret);
     registerSubsystem((SubsystemBase)turret, turretControls);
 
-    widgets = new ShuffleboardWidgets();
-    widgets.register();
+    //widgets = new ShuffleboardWidgets();
```
Comment (r-wallis) [inline src/main/java/frc/robot/Robot.java]: put widgets back

File: src/main/java/frc/robot/commands/TurretControls.java [hunk 3]
```diff
@@ -38,7 +38,8 @@ public void execute() {
     // double angle = Math.atan2(targetY - robotY, targetX - robotX);
     double angle = OI.driverController.getRawAxis(0)*10;
     // double adjustedAngle = angle - Robot.drivetrain.getAngleRadians().getRadians();
-    Robot.turret.setPosition(angle);
+    
+    // Robot.turret.setPosition(angle);
```
Comment (r-wallis) [inline src/main/java/frc/robot/commands/TurretControls.java]: put turret back...no robot.

File: src/main/java/frc/robot/shuffleboard/ShuffleboardWidgets.java [hunk 4]
```diff
@@ -105,12 +106,15 @@ public ShuffleboardWidgets() {
...
-    drivetrain = Robot.drivetrain;
-    turret = Robot.turret;
-    shooter = Robot.shooter;
-    magazine = Robot.magazine;
-    lift = Robot.lift;
-    winch = (WinchInterface) Robot.drivetrain;
+    this.drivetrain = drivetrain;
+    this.turret = turret;
+
+    // drivetrain = Robot.drivetrain;
```
Comment (r-wallis) [inline src/main/java/frc/robot/shuffleboard/ShuffleboardWidgets.java]: remove the commented out stuff

File: src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (r-wallis) [reply src/main/java/frc/robot/Robot.java]: done

**Review by r-wallis — DISMISSED** [review]

File: src/main/java/frc/robot/subsystems/instances/Drivetrain.java [hunk 5]
```diff
@@ -254,9 +254,9 @@ public void engageDrivetrain() {
         rightMotorLeader.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
         leftMotorLeader.setSensorPhase(true);
         rightMotorLeader.setSensorPhase(true);
-        double P = 0.2;
-        double I = 0.005;
-        double D = 8;
+        double P = 0.05;
```
Comment (jjesse1972) [inline src/main/java/frc/robot/subsystems/instances/Drivetrain.java:257]: Is this intentional

Comment (jjesse1972) [reply src/main/java/frc/robot/subsystems/instances/Drivetrain.java:257]: Ben confirm that this is intensional.

**Review by jjesse1972 — APPROVED** [review]

**Review by Silvery-Sheen — APPROVED** [review]

**Pull request merged**